Splunk_TA_paloalto/default/eventtypes.conf

```diff
@@ -14,7 +14,7 @@ search = sourcetype=pan_config OR sourcetype=pan:config
 #tags = change
 
 [pan_traffic]
-search = sourcetype=pan_traffic OR sourcetype=pan:traffic OR (sourcetype=pan:firewall_cloud AND LogType="TRAFFIC")
+search = sourcetype=pan_traffic OR sourcetype=pan:traffic OR (sourcetype=pan:firewall_cloud AND LogType="TRAFFIC") AND log_subtype != "start"
```

Comment:
I understand the need to switch from Session datamodel to Traffic datamodel, but why do we need to make `pan_traffic` refer only to 'end' logs?

Reply:
We have mapped the start events with the Network Session Data Model. Because of this, two data models were getting mapped in start events. So, we excluded the start events from the search query. But, as discussed, we are not going to map the session data model for the start events, we can remove the condition for log_subtype != "start" from the query.

Reply:
Excellent, thanks! In the merge I changed pan_traffic back to start and end, but removed tags for it and pan_traffic_start. Now only pan_traffic_end gets tagged and it goes to the Network Traffic datamodel. So I think this one is resolved.